README.md

```diff
@@ -9,388 +11,17 @@
 
 > Sentry module for Nuxt.js
 
+<p align="center">
+  <a href="https://sentry.nuxtjs.org">Read Documentation</a>
+</p>
+
 ## Features
 
 The module enables error logging through [Sentry](http://sentry.io).
 
 - **Please note** that version 2.2.0 of this package removed the older `public_key` and `private_key` options, since the updated Sentry packages don't support these anymore.
 - **Please note** that version 2.0.0 of this package introduces a breaking change. See [#30](https://github.com/nuxt-community/sentry-module/pull/30) for more information.
```

Comment:
Let's just remove those notes now. Both from here and documentation. Those are very outdated.